packages/ui-components/src/components/Pagination/Pagination.test.js

```diff
@@ -110,6 +216,17 @@ describe("Pagination", () => {
     })
   })
 
+  test("fires onKeyPress handler on Enter as passed for input variant with undefined controlPage", async () => {
```

Comment:
same again, test checks that handler is not called in this case.
```suggestion
  test("does not fire onKeyPress handler on Enter for input variant with undefined controlPage", async () => {
```